bundles/pom.xml

```diff
@@ -236,7 +238,7 @@
     <module>org.openhab.binding.snmp</module>
     <module>org.openhab.binding.solaredge</module>
     <module>org.openhab.binding.solarlog</module>
-	<module>org.openhab.binding.somfymylink</module>
+    <module>org.openhab.binding.somfymylink</module>
```

Comment:
Can you leave this as-is?

Reply:
Sorry for this but this must somewhere some configuration problem in my git explorer because I don't see this in my offline version only in the online version

Reply:
You can enable/disable ignoring of whitespaces in some Git UIs.

Reply:
resolved

Reply:
Here, too.